pkg/templates/templates/credentials/create/credential-set.schema.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
Sorry for the confusion around what this template should return! If you have any improvements over the existing schema https://github.com/getporter/porter/blob/release/v1/pkg/schema/credential-set.schema.json, definitely add them to that file!

This template should be just the json version of the yaml file (obviously without comments):

```json
{
  "schemaType": "CredentialSet",
  "schemaVersion": "1.0.0",
  "name": "NAME",
  "namespace": "NAMESPACE",
  "labels": {
    "LABEL": "LABEL_VALUE"
  },
  "credentials": [
    {
      "name": "CREDENTIAL_NAME",
      "source": {
        "env": "CREDENTIAL_VALUE"
      }
    }
  ]
}
```

Reply:
Ah no, this should be my fault, I am sorry. Looking at it now, I am not sure why did I use the placeholders for the `.yaml` template but using the different format for the `.json` file back then 🤔 Thank you for the really good catch, Carolyn! Will fix it at the next commits.